src/ui/templates/search/search.hbs

```diff
@@ -49,12 +49,12 @@
           data-opts='{ "iconUrl": "{{_config.customIconUrl}}" }'
       ></div>
     {{else}}
-      <div class="js-yxt-AnimatedForward yxt-SearchBar-AnimatedIcon--paused
+      <div class="yxt-SearchBar-AnimatedIcon js-yxt-AnimatedForward yxt-SearchBar-AnimatedIcon--paused
```

Comment:
can we have the `yxt-SearchBar-AnimatedIcon` and `yxt-SearchBar-AnimatedIcon--paused` right next to each other, with latter after the former. it makes it easier to know the effects when trying to determine how `--paused` overrides the base `AnimatedIcon` class